### PR TITLE
PDF static content

### PR DESCRIPTION
**Pull request by goose-life**
This adds support for making running headers and footers translatable and easier to override.

### CODE REVIEW

File: indigo_api/templates/indigo_api/akn/export/pdf_static_content_akn.xml
Comment (longhotsummer) [inline indigo_api/templates/indigo_api/akn/export/pdf_static_content_akn.xml:1]: Does this need to be `.xml.html` for makemessages to work?

Comment (goose-life) [reply indigo_api/templates/indigo_api/akn/export/pdf_static_content_akn.xml:1]: just adding `-e xml` works — let me run that and add it to this PR

File: indigo_api/exporters.py [hunk 1]
```diff
@@ -202,6 +203,41 @@ def html_to_xml(self, html_string):
...
+            'place_string': self.get_place_string(document),
+        }
+
+    def get_place_string(self, document):
+        """ Get the appropriate string based on the locality and country, in the right language.
+        """
+        locality = document.work.locality.name if document.work.locality else None
+        country = document.work.country.name if document.country not in self.dont_include_countries else None
+        if locality and country:
+            return _('%(locality), %(country)s') % {'locality': _(locality), 'country': _(country)}
```
Comment (longhotsummer) [inline indigo_api/exporters.py:235]: I'm not sure of the value of `(%s, %s)` being translateable, but it won't hurt.

Comment (goose-life) [reply indigo_api/exporters.py]: eh yeah this was silly, I got rid of it 😅

**Review by longhotsummer — APPROVED** [review]

**Pull request merged**